Fix bug in SI set_vacuum_chamber

xresende opened the pull request:
This bug has been lurking for almost four years now, when model `SI_V20_01` was created ... 
It was introduced in commit 116daf7886f0ea9dda05e6fe12d851a82a5377ce when the model was shifted in order to introduce physical limits for ID vacuum chambers. The shifting function creates a new `the_ring` which was not passed back to the main `create_lattice` function.

Although this bug affects only SI model, I standardized the code for the other accelerators.

fernandohds564 commented:
I'm  not sure about the origin of this bug, @xresende. I think it is new and I introduced it when I changed the behavior of pyaccel in some refactoring I have done recently....
I think might worked before, because a copy, at least a copy of the elements, was not done, so the return of the ring model was not needed.
I'm not sure about this either.... For now it is just a feeling.

xresende commented:
> I'm not sure about the origin of this bug, @xresende. I think it is new and I introduced it when I changed the behavior of pyaccel in some refactoring I have done recently....
> I think might worked before, because a copy, at least a copy of the elements, was not done, so the return of the ring model was not needed.
> I'm not sure about this either.... For now it is just a feeling.

I understand. You might be right: it may have been introduced when trackcpp python lib was changed. Anyway, I think we did not use `pyaccel` for any calculation from which _specs_ were defined, so the bug probably did not compromise anything. Even the study of IDs was performed mainly with matlab AT, I think.

xresende commented:
- I have commented out ID vacuum chambers or now.
- fixed `hmax` in injection section.
- added 4 mm vchamber limits also in marks in high field BC regions.
